docs/usage.rst

```diff
@@ -77,6 +77,8 @@ General:
         When set, use the value to answer the passphrase question for encrypted repositories.
     BORG_DISPLAY_PASSPHRASE
         When set, use the value to answer the "display the passphrase for verification" question when defining a new passphrase for encrypted repositories.
+    BORG_KEY_FILE
+        When set, use the given filename as repository key file.
```

Comment:
There is a later section "Directories" where BORG_KEYS_DIR is. Rename that section to "Directories and files" and move this there?